### DISCUSSION TITLE
Require Full flash theme(Except the theme module) on Razor based instead of liquid - Razor Pages documentation / example project

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10634] [Q&A]
Require Full flash theme Like we have the in the Blog theme see attachment below:

![missing theme](https://user-images.githubusercontent.com/5202291/140423302-e566c594-e35c-4895-963f-6e7d713a42b5.png)

Also anyone please share with me the full flash theme link to download code based on razor based. Need to develop the decouple cms end to end application.

**Comment by Piedone** [comment]
What exactly do you mean by "full flash"? To enforce that a theme includes some basic views?

Note that what's in TheBlogTheme is not complete at all, and a lot of that is specific to how it sets up the site with the recipe in the theme. E.g. a Content-Article template is only useful if you have a content type called Article.

**Reply by saleemhaq** [reply]
> What exactly do you mean by "full flash"? To enforce that a theme includes some basic views?
> 
> Note that what's in TheBlogTheme is not complete at all, and a lot of that is specific to how it sets up the site with the recipe in the theme. E.g. a Content-Article template is only useful if you have a content type called Article.

Full flash mean like a repo on github: https://github.com/psijkof/ModernBusiness.OC.RazorPages in Razor pages but this project is refuse to build ?? so that kind of full flash repo I required from where I see how pages working in proper flow in decouple CMS. I m using because that is required. Can you share me any repository so it help me out.

**Reply by Piedone** [reply]
Oh, you mean "full source"? That project you linked and the following resources should get you going:

- https://docs.orchardcore.net/en/dev/docs/guides/decoupled-cms/
- https://docs.microsoft.com/en-us/aspnet/core/razor-pages/
- https://www.youtube.com/watch?v=yWpz8p-oaKg
